### PR TITLE
upgrade to python3.11 and nodejs 18.x

### PR DESCRIPTION
**Pull request by PhuNguyenMTT**
Update python version 3.7->3.11 and nodejs 14.x -> 18.x
Lib python upgraded:

- boto3==1.27.1
- botocore==1.30.1
- certifi==2023.7.22
- chardet==5.2.0
- charset-normalizer==3.3.2
- crhelper==2.0.11
- idna==3.4
- mypy-boto3-cloudformation==1.28.64
- mypy-boto3-config==1.28.78
- mypy-boto3-iam==1.28.79
- mypy-boto3-organizations==1.28.36
- mypy-boto3-securityhub==1.28.61
- mypy-boto3-sns==1.28.72
- mypy-boto3-sts==1.28.58
- requests==2.31.0
- retrying==1.3.4
- six==1.16.0
- typing_extensions==4.8.0
- urllib3==1.26.18
- launchpadlib==1.11.0

Upgrade AWS::Serverless::Application SemanticVersion for securityhub.template.yaml

### CODE REVIEW

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+mypy-boto3-iam==1.28.70
+mypy-boto3-organizations==1.28.36
+mypy-boto3-securityhub==1.28.61
+mypy-boto3-sns==1.28.36
+mypy-boto3-sts==1.28.58
+requests==2.31.0
+retrying==1.3.4
+six==1.16.0
+typing_extensions==4.6.1
+urllib3==1.26.15
```
Comment (npande) [inline apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: Can urllib3 be upgraded to 2.0.7 please. With Python v3.11, it is possible to use urllib3 v2.0.7

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+boto3==1.27.1
+botocore==1.30.1
+certifi==2023.5.7
```
Comment (npande) [inline apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: Can certifi be upgraded to 2023.7.22 please.

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+mypy-boto3-organizations==1.28.36
+mypy-boto3-securityhub==1.28.61
+mypy-boto3-sns==1.28.36
+mypy-boto3-sts==1.28.58
+requests==2.31.0
+retrying==1.3.4
+six==1.16.0
+typing_extensions==4.6.1
+urllib3==1.26.15
+launchpadlib==1.10.13
```
Comment (npande) [inline apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: Can launchpadlib be upgraded to 1.11.0 please.

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
+boto3==1.27.1
+botocore==1.30.1
+certifi==2023.5.7
+chardet==5.2.0
+charset-normalizer==3.3.1
```
Comment (npande) [inline apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: In general check if you can upgrade to latest version available and applicable today for all the python modules.

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt (hunk 1 above)
Comment (PhuNguyenMTT) [reply apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: ERROR: botocore 1.30.1 has requirement urllib3<1.27,>=1.25.4, but you'll have urllib3 2.0.7 which is incompatible.
1.26.18 is latest which is compatible

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt [hunk 5]
```diff
@@ -1,20 +1,20 @@
 boto3==1.27.1
 botocore==1.30.1
```
Comment (npande) [inline apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt:2]: can botocore be upgraded to 1.31.80

Comment (PhuNguyenMTT) [reply apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt:2]: photo_2023-11-08_17-57-17
1.27.1 is version that aws recommended for python3.11

Comment (PhuNguyenMTT) [reply apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt:2]: anyway, botocore-1.30.1

File: apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt (hunk 4 above)
Comment (PhuNguyenMTT) [reply apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt]: I have checked all python modules, and then upgraded it

**Review by npande — APPROVED** [review]
LGTM

File: apps/sumo-aws-apps/security-hub/securityhub.template.yaml [hunk 6]
```diff
@@ -419,15 +419,15 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:956882708938:applications/sumologic-securityhub-forwarder
-        SemanticVersion: 1.0.7
+        SemanticVersion: 1.0.8
```
Comment (himanshu219) [inline apps/sumo-aws-apps/security-hub/securityhub.template.yaml]: sumologic-securityhub-connector-aws-org 1.0.4
sumologic-securityhub-forwarder 1.0.10
sumologic-securityhub-collector 1.0.9
Please update to latest versions in both the repos.

Comment (PhuNguyenMTT) [reply apps/sumo-aws-apps/security-hub/securityhub.template.yaml]: upgraded sumologic-securityhub-forwarder 1.0.10
sumologic-securityhub-collector 1.0.9

**Review by PhuNguyenMTT — COMMENTED** [review]
upgraded 
sumologic-securityhub-forwarder 1.0.10
sumologic-securityhub-collector 1.0.9

**Pull request merged**